.github/ISSUE_TEMPLATE/proposal.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Protocol Change Proposal
+about: Create a proposal to request a change to the protocol
+
+---
+
+<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
+v                            ✰  Thanks for opening an issue! ✰    
+v    Before smashing the submit button please review the template.
+v    Word of caution: poorly thought-out proposals may be rejected 
+v                     without deliberation 
```

Comment:
Can we make this a tad gentler? Maybe something like `Under-specified proposals may be rejected summarily`? I don't want people to think we won't think at all about their proposals